src/System.Runtime/tests/CompareToTests.netstandard1.7.cs

```diff
@@ -0,0 +1,165 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class CompareToTests
+    {
+        [Fact]
+        public static void FromBoolean()
+        {
+            object testValue = false;
+            object testValue2 = true;
+            object wrongValue = "hello";
+            bool val1 = false;
+            bool val2 = true;
+            Assert.Equal(0, val1.CompareTo(testValue));
```

Comment:
+1, that is much more readable.

---
In reply to: [76328953](https://github.com/dotnet/corefx/pull/11134#discussion_r76328953) [](ancestors = 76328953)